packages/frontend/components/Borrow/Wrapper.tsx

```diff
@@ -100,14 +100,10 @@ function BorrowWrapper({ query }: BorrowWrapperProps) {
   return (
     <>
       <Head>
-        <title>{`${isEditing ? 'Position' : 'Borrow'} - xFuji`}</title>
+        <title>Position - Fuji V2 Himalaya</title>
```

Comment:
With this change, the browser will show "Position" as the title instead of "Borrow" when the user creates a new position, which is super confusing. We're trying to distinguish between them as much as possible. I think we need to change it @brozorec .

Reply:
That's true, agree to revert